### ISSUE TITLE
Cannot GET /live/STREAM_KEY/index.m3u8

### ISSUE THREAD

**Issue by TheDude53** [illuspas/Node-Media-Server#282]
I too have this issue however the other posted issues seem to be outdated. No idea where I went wrong since I just ran `npm i node-media-server`, put the example code from the README in a js file and ran it with `node server.js`, using OBS Studio via RTMP to post stream. I am running Windows 10 x64 which is also a reason I am reposting, since the other issues were for Ubuntu/Linux. Any help would be greatly appreciated!

**Comment by rrfaria** [comment]
Create a vm using ubuntu and it will be easier to you. I did that

**Comment by TheDude53** [comment]
I have a low spec computer and any VM that uses more RAM than 768MB will make the computer freeze + if the VM does work it will be gruelingly slow. Is it possible to do the Windows version?

**Comment by sakkatos** [comment]
I've also get this issue. It's seem because  path file of index.m3u8 is the problem. 

First, in source code when they construct path file they just concatenate string rather than use path.join(). So if you fix this you will find file m3u8 in somewhere same as project folder's drive. For excample you expect file should be at D:\ProjectFolder\Media\Live\MyStream\index.m3u8 you might find it at D:\ProjectFolderMediaLiveMyStreamindex.m3u8.   

Second, I don't because of I run this on windows but you have to escape backslash ('\') in path file before send to arguments of command line. 

In short, I fixed this by go to node_modules\node-media-server\node_trans_session.js and add this line of code `mapStr = mapStr.replace(/\\/g, '\\\\')` before this line `Array.prototype.push.apply(argv, ['-f', 'tee', '-map', '0:a?', '-map', '0:v?', mapStr]);`

** warning ** I understand what I'm doing. Use it at your own risk.
